pkg/pipelines/tekton/tasks.go

```diff
@@ -303,7 +303,7 @@ spec:
       description: Digest of the image just built.
   steps:
     - name: generate
-      image: quay.io/boson/s2i:latest
```

Comment:
ugh, how old was that? :sweat_smile:

Reply:
Quite. Still much up to date that we use now in midstream.

Reply:
`quay.io/boson/s2i:latest` is our image we build manually once in blue moon.